### ISSUE TITLE
Hugging-face online model not found when decoding Diffusion-LM

### ISSUE THREAD

**Issue by Junyi42** [XiangLi1999/Diffusion-LM#18]
Hey there, I was just trying to run the decoding model while it says I have an error about 

`OSError: We couldn't connect to 'https://huggingface.co/' to load this model and it looks like predictability/diff_models/e2e-tgt_e=15_b=20_m=gpt2_wikitext-103-raw-v1_101_None is not the path to a directory conaining a config.json file.`

Just 10 hours ago I was able to successfully load the model from hugging-face, I wonder if this is my personal problem?

**Comment by XiangLi1999** [comment]
no... it's not your problem. I didn't post the evaluation model on hugging face hub. But you can easily train your own AR model for evaluation.

**Comment by XiangLi1999** [comment]
I think the fine-tuning code is also available in the git repo.

**Comment by lipingtang17** [comment]
Hi Lisa, I met the same problem. Could you please upload your evaluation model? Thanks!

**Comment by YangYu-NUAA** [comment]
model_name_or_path='predictability/diff_models/compress_e=5_b=60_m=gpt2_wikitext-103-raw-v1_None

Sorry for this quetion, but i really want to know the function of this evalution model so i can train a similar model to replace it. Now i'm confusing about this pretrained model.

**Comment by aatmikg** [comment]
Hi Lisa, would it be possible to share any information on the hyperparameters used to train the AR models / any instructions so we can replicate your training process? You noted that fine-tuning code is included in the repo, if so, could you point us to where we might find it? I tried to look but couldn't figure it out.

Any help would be greatly appreciated, thanks!

**Comment by rzhwang** [comment]
> model_name_or_path='predictability/diff_models/compress_e=5_b=60_m=gpt2_wikitext-103-raw-v1_None
> 
> Sorry for this quetion, but i really want to know the function of this evalution model so i can train a similar model to replace it. Now i'm confusing about this pretrained model.

have
